src/busterbunny.js

```diff
@@ -212,9 +212,16 @@ module.exports = (function() {
                 config.queues.forEach(function(queue) {
                     channel.assertQueue(queue.name);
                     channel.consume(queue.name, function(message) {
+                        var event;
                         try {
-                            var event = self.encoder.decodeEvent(message);
+                            event = self.encoder.decodeEvent(message);
+                        } catch(err) {
+                            channel.nack(message, false, false);
+                            _stats.messagesRejectedWithRetry++;
+                            self.emit(self.EVENTS.EVENT_NACKED, null, message, false, new Date().getTime());
+                        }
 
+                        if(event) {
```

Comment:
A try/catch will not catch things if they are done asynchronously. The reject/ack functions are called asynchronously so the only use try/catch has is for catching variable assignment and if self.emit fails (which if it does then everything else should fail).